### PR TITLE
Add video paper builder fargate app

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

**Comment by dougmartin** [comment]
@scytacki this PR does **not** refactor the common templates with the `log-puller` app.  That will be a separate PR later.

File: fargate/video-paper-builder.yml [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+          - ''
+          - - !Ref Subdomain
+            - '.'
+            - Fn::ImportValue: !Join [':', [!Ref 'StackName', 'HostedZoneName']]
+      Type: A
+      AliasTarget:
+        DNSName:
+          Fn::ImportValue: !Join [':', [!Ref 'StackName', 'LoadBalancerDNSName']]
+        HostedZoneId:
+          Fn::ImportValue: !Join [':', [!Ref 'StackName', 'LoadBalancerCanonicalHostedZoneID']]
```
Comment (scytacki) [inline fargate/video-paper-builder.yml:415]: I'd still recommend adding the weighted config here. So you can later migrate this DNSRecord to another template if necessary.

Comment (dougmartin) [reply fargate/video-paper-builder.yml:415]: I plan on addressing that in the refactoring of the common templates

Comment (scytacki) [reply fargate/video-paper-builder.yml:415]: OK, it is just that you won't be able to refactor without doing it first in this template, before moving the entry to another template.

**Pull request merged**